index.html

```diff
@@ -222,20 +222,25 @@
                     data-split-options='{"type":"lines"}'
                   >
                     <span class="ld-fh-txt">
-                      Bitcoin is designed around the idea of using money.</span
-                    >
+                      HACKERS' CLUB </span>
```

Comment:
Don't use full block letters. Use `Start Case`: Hackers' Club University of Peradeniya

Reply:
done